### DISCUSSION TITLE
ovos events - where are the docs

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#230] [Q&A] [answered]
https://openvoiceos.github.io/ovos-technical-manual/search.html?q=events

lists no events.

i have my legacy code:
        self.add_event( "mycroft.skills.initialized", self.handle_boot_finished )

and need to know how to do it the ovos way

**Comment by JarbasAl** [comment, marked as answer]
It is done the exact same way

dedicated docs here https://openvoiceos.github.io/ovos-technical-manual/skills_bus/
